src/plugins/ui_actions/public/cell_actions/components/cell_actions_context.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { orderBy } from 'lodash/fp';
+import React, { createContext, FC, useContext, useMemo } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import useAsyncFn from 'react-use/lib/useAsyncFn';
+import type { Action } from '../../actions';
+import { CellActionExecutionContext } from './cell_actions';
+
+type GetActionsType = (trigger: string, context: object) => Promise<Action[]>;
```

Comment:
Can't we define a type for the `context`? It will be something like `{ field: string, value: unknown, fieldType: string }` right?
(I had to turn it into an array of objects on the lens actions, but we can ignore that for the alpha)

Reply:
I think it is the same as defined in `CellActionField`. We may also need to pass `eventId: string` eventually, it is the id of the event/alert, needed by the _AddToTimeline_ action.
Maybe we should call this type `CellActionData` or `CellActionPayload`, to not use _Field_ or _Context_ again